packages/react/src/index.js

```diff
@@ -9,15 +9,20 @@ export { default as css } from './css'
 
 if (process.env.NODE_ENV !== 'production') {
   const isBrowser = typeof document !== 'undefined'
-  const globalKey = '__EMOTION_REACT__'
-  const globalContext = isBrowser ? window : global
-  if (globalContext[globalKey]) {
-    console.warn(
-      'You are loading @emotion/react when it is already loaded. Running ' +
-        'multiple instances may cause problems. This can happen if multiple ' +
-        'versions are used, or if multiple builds of the same version are ' +
-        'used.'
-    )
+  // #1727 for some reason Jest evaluates modules twice if some consuming module gets mocked with jest.mock
+  const isJest = typeof process !== 'undefined' && process.env.JEST_WORKER_ID
```

Comment:
Why not `typeof jest !== 'undefined'`? I'm afraid that using `process` in a webpack build would pull in a bunch of code

Reply:
This was suggested in the issue and I forgot that this can be simplified like this 😅 changed it right now to what you have suggested